ui/rewards/dashboard/RewardsDashboardCard.tsx

```diff
@@ -11,38 +12,48 @@ type Props = {
   availableSoon?: boolean;
   blurFilter?: boolean;
   contentAfter?: React.ReactNode;
-  direction?: 'column' | 'column-reverse' | 'row';
   reverse?: boolean;
   children?: React.ReactNode;
+  label?: string;
+  isLoading?: boolean;
+  cardValueStyle?: object;
+  className?: string;
 };
 
 const RewardsDashboardCard = ({
-  title, description, hint, availableSoon, contentAfter,
-  direction = 'column', children, blurFilter,
+  title, description, availableSoon, contentAfter, cardValueStyle, hint,
+  children, blurFilter, label, isLoading, className,
 }: Props) => {
   return (
     <Flex
-      flexDirection={{ base: direction === 'row' ? 'column' : direction, md: direction }}
-      justifyContent={ direction === 'column-reverse' ? 'flex-end' : 'flex-start' }
+      flexDirection="column"
+      justifyContent="flex-start"
       p={{ base: 1.5, md: 2 }}
       border="1px solid"
       borderColor={{ _light: 'gray.200', _dark: 'whiteAlpha.200' }}
       borderRadius="lg"
-      gap={{ base: 1, md: direction === 'row' ? 10 : 1 }}
-      w={ direction === 'row' ? 'full' : 'auto' }
-      flex={ direction !== 'row' ? 1 : '0 1 auto' }
+      gap={ 1 }
+      flex={ 1 }
+      className={ className }
     >
       <Flex
         flexDirection="column"
         gap={ 2 }
         p={{ base: 1.5, md: 3 }}
-        w={{ base: 'full', md: direction === 'row' ? '340px' : 'full' }}
+        w="full"
       >
-        <Flex alignItems="center" gap={ 2 }>
-          <Text fontSize={{ base: 'md', md: 'lg' }} fontWeight="500">{ title }</Text>
-          { hint && <Hint label={ hint }/> }
-          { availableSoon && <Badge colorPalette="blue">Available soon</Badge> }
-        </Flex>
+        { label && (
+          <Skeleton loading={ isLoading } w="fit-content">
+            <Badge>{ label }</Badge>
+          </Skeleton>
+        ) }
+        { title && (
+          <Flex alignItems="center" gap={ 2 }>
+            <Text fontSize={{ base: 'md', md: 'lg' }} fontWeight="500">{ title }</Text>
```

Comment:
```suggestion
            <Text textStyle={{ base: 'md', md: 'lg' }} fontWeight="500">{ title }</Text>
```

Instead of `fontSize` please use `textStyle` prop. And we don't have `lg` text size anymore - https://www.figma.com/design/4In0X8UADoZaTfZ34HaZ3K/Blockscout-design-system?node-id=24665-6229&m=dev&t=wm1D2MHsyXVPH0tz-1